src/pages/settings/server/Members.tsx

```diff
@@ -74,7 +74,7 @@ const Inner = observer(({ member }: InnerProps) => {
                         );
                     })}
                     <Button
-                        compact
+                        palette="secondary"
```

Comment:
Slight colour change, albeit nothing blocking (and in my opinion an improvement):

Nightly:

![image](https://user-images.githubusercontent.com/42586271/169715932-38d0d28d-2ad1-4d4e-9ce9-5c64c8ba6eb2.png)

---

Branch:

![image](https://user-images.githubusercontent.com/42586271/169715923-48221636-7b81-471b-be44-0a5b0faaeedf.png)

Reply:
We've decided to stick with the new button